sample.env

```diff
@@ -4,8 +4,9 @@
 CCDC_LICENSING_CONFIGURATION=la-code;123456-123456-123456-123456-123456-123456;
 
 # Password for local database
-CSD_DB_PASSWORD=A password of your choosing
-CSD_CACHE_PASSWORD=A password of your choosing
+CSD_DB_CONNECTIONSTRING=Server=database-server;Port=5432;Database=csd-database;User Id=postgres;Password=passwordhere
```

Comment:
There is another branch here that manages database config, did we want to use it? or forget it?

https://github.com/ccdc-opensource/on-site-webcsd/blob/generic-database-config/sample.env

Reply:
@hudson-ccdc could we have a chat about this as I'm unsure what your trying to say

Reply:
I have deleted the branch